thoth/package_extract/image.py

```diff
@@ -178,6 +179,24 @@ def _parse_deb_dependency_line(line_str: str) -> typing.List[tuple]:
     return result
 
 
+def _gather_digests(path: str) -> typing.List[dict]:
```

Comment:
Should I used just ```list``` instead of ```typing.List[dict]``` as a return type?

Reply:
`typing.List[dict]` is better :+1:, it specifies more what is returned.